File: TellDontAsk/AlarmsProcedural/Adapters/TimeService.cs
using TddEbook.TddToolkit;

namespace AlarmsProcedural.Adapters
{
  static class TimeService
  {
    public static bool IsWeekend()
    {
      return Any.Boolean();
    }

    public static bool IsOutsideWeekend()
    {
      return !IsWeekend();
    }

    public static bool IsNight()
    {
      return Any.Boolean();
    }

    public static bool IsDay()
    {
      return !IsNight();
    }
  }
}

File: Java/AlarmsProcedural/src/main/java/Adapters/TimeService.java
package Adapters;

import autofixture.publicinterface.Any;

public class TimeService {
  public static boolean isWeekend() {
    return Any.instanceOf(boolean.class);
  }

  public static boolean isOutsideWeekend() {
    return !isWeekend();
  }

  public static boolean isNight() {
    return Any.instanceOf(boolean.class);
  }

  public static boolean isDay() {
    return !isNight();
  }
}
